SPQR/Source/spqr_parallel.cpp

```diff
@@ -99,28 +99,27 @@ template <typename Entry, typename Int> void spqr_parallel
         spqr_zippy <Entry, Int> (ntasks-1, Blob) ;
     task::spawn_root_and_wait (a) ;
 }
+
+
+// explicit instantiations
+
 template void spqr_parallel <double, int32_t>
 (
-    int32_t ntasks,
```

Comment:
These changes are also spurious and not my style

Reply:
Is there a style-guide? What would be the correct style?

Imho, these explicit instantiations of templates are just a technicality and can be short. That makes it easier to see on the same screen for which types a template is explicitly instantiated.